src/internal/_none.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function _none(fn, list) {
+    var idx = -1;
+    while (++idx < list.length) {
+        if (fn(list[idx])) {
+            return false;
+        }
+    }
+    return true;
+};
```

Comment:
There's no need for this internal function, since it's only used in the definition of `R.none`.